src/scenes/app/stores/index.js

```diff
@@ -1,38 +1,65 @@
-import { observable, action } from 'mobx';
-import sampleData from '../../../helpers/dataSourceSample';
+import { observable, action, toJS } from 'mobx';
 import strings from '../../../localization/en/strings';
+import Utils from '../../../helpers/Utils';
 
 class ObservableTodoStore {
-  @observable items = sampleData;
+  @observable items = [];
   @observable error = '';
+  @observable start = true;
 
   @action
   clearAllDone = () => {
-    this.items = this.items.filter(element => element.done === false);
+    this.items = this.items.filter(element => element.completed === false);
+  }
+
+  getToDosDone = () => {
```

Comment:
This could be a `computed` function.

Reply:
![image](https://user-images.githubusercontent.com/26419582/44541985-06b2c900-a6e2-11e8-8302-f1a211dc1012.png)

Reply:
I tried..